### PR TITLE
task text overflow fix

### PR DESCRIPTION
**Pull request by leninmeher**
Bug: #5552

Adding a task with a long text containing a long first line isn now truncated

Screenshot 2024-05-24 at 12 56 45 AM

### CODE REVIEW

**Comment by charlesBochet** [comment]
Hi @leninmeher, I don't think this strategy works. I have investigated this kind of issue previously and we should solve it using flex properly. Applying, width percentage won't give a result accurate enough (we will always either lose space or it will be too wide depending on user screen).

Also, this seems broken when we have no description
image

If you want to give a try, I have posted a technical strategy on similar issue here: https://github.com/twentyhq/twenty/pull/5360

In a nutshell, we have containers that cannot shrink and some that can shrink, flex allows us to handle this properly. It's a bit tricky to adjust properly but it's a good opportunity do dive into flex behaviors

**Comment by FelixMalfait** [comment]
@leninmeher closing this PR as there's no activity but let us know if we should re-open it! Thanks
